_data/projects/angular2.yml

```diff
@@ -0,0 +1,10 @@
+name: Angular2
+desc: HTML enhanced for web apps
+site: https://angular.io
+tags:
+- JavaScript
+- web
+- Angular 2
+upforgrabs:
+  name: hotlist: community-help
```

Comment:
The `:` here is causing a parsing issue. You'll need to wrap this value in double-quotes.